scan_pro_robot:
  arm_controller:
      type: position_controllers/JointTrajectoryController
      joints:
        - joint1
        - joint2
        - joint3
        - joint4
        - joint5
        - joint6
      # constraints:
        # goal_time: 0.6
        # stopped_velocity_tolerance: 0.05
        # hip: {trajectory: 0.1, goal: 0.1}
        # shoulder: {trajectory: 0.1, goal: 0.1}
        # elbow: {trajectory: 0.1, goal: 0.1}
        # wrist: {trajectory: 0.1, goal: 0.1}
      # stop_trajectory_duration: 0.5
      # state_publish_rate:  25
      # action_monitor_rate: 10
  /gazebo_ros_control:
      pid_gains:
        joint1: {p: 1000.0, i: 100.0, d: 10.0}
        joint2: {p: 1000.0, i: 100.0, d: 10.0}
        joint3: {p: 1000.0, i: 100.0, d: 10.0}
        joint4: {p: 1000.0, i: 100.0, d: 10.0}
        joint5: {p: 1000.0, i: 100.0, d: 10.0}
        joint6: {p: 100.0, i: 0.0, d: 0.0}
